Recursion/C++/towers_of_honai.cpp

```diff
@@ -0,0 +1,23 @@
+/* Tower of Hanoi is a mathematical puzzle where we have three rods and n disks. The objective of the puzzle is to move the entire stack to another rod */
+#include<iostream>
+using namespace std;
+void toh(int n,char sou,char aux,char des);          //function declaration
+int main(){
+	int n;
+	cout<<"enter the number of disks "<<endl;
+	cin>>n;
+	toh(n,'A','B','C');                           //functional call
+	return 0;
+}
+void toh(int n,char sou,char aux,char des){                              //function definition
```

Comment:
please explain this function with more comments . Its not understandable for a beginner

Reply:
I have added few comments to explain the function clearly